src/Stores/Cases/actions.js

```diff
@@ -26,9 +33,31 @@ export const fetchCaseById = (id: CaseId) => (dispatch: Dispatch) => {
   );
 };
 
+/* 
+    GET ALL CASES
+*/
+
+const getCases = () => ({ type: actionTypes.GET_CASES_REQUEST });
+
+const getCasesSuccess = (cases: Array<Case>) => ({
+  cases,
+  type: actionTypes.GET_CASES_SUCCESS,
+});
+
+const getCasesFailure = (error: *) => ({
```

Comment:
the type here is just String no?
`const getCasesFailure = (error: string) => ({`

Reply:
you are correct!